core/arch/arm/kernel/sub.mk

```diff
@@ -48,6 +48,7 @@ ifeq ($(CFG_GENERIC_BOOT),y)
 srcs-$(CFG_ARM32_core) += generic_entry_a32.S
 srcs-$(CFG_ARM64_core) += generic_entry_a64.S
 endif
+srcs-y += initcall.c
```

Comment:
**initcall.c** is not arm specific? Move to **core/kernel/**?

Reply:
True, I'll move it.